bigchaindb/core.py

```diff
@@ -43,19 +38,11 @@ def __init__(self, public_key=None, private_key=None, keyring=[], connection=Non
         its default value (defined in bigchaindb.__init__).
 
         Args:
-            public_key (str): the base58 encoded public key for the ED25519 curve.
-            private_key (str): the base58 encoded private key for the ED25519 curve.
-            keyring (list[str]): list of base58 encoded public keys of the federation nodes.
             connection (:class:`~bigchaindb.backend.connection.Connection`):
                 A connection to the database.
         """
-
         config_utils.autoconfigure()
 
-        self.me = public_key or bigchaindb.config['keypair']['public']
-        self.me_private = private_key or bigchaindb.config['keypair']['private']
```

Comment:
I did a grep search for any other mentions of `me_private` and found many (too many to copy here), all of which were in the `tests/` directory. I guess the associated tests aren't failing because they aren't being run.

Reply:
Yes, I can update/remove them, depending on the state of those test w.r.t 2.0.